google-cloud-tools-plugin/src/com/google/gct/idea/appengine/cloud/ManagedVmCloudConfigurable.java

```diff
@@ -153,9 +148,20 @@ public boolean isModified() {
 
   @Override
   public void apply() throws ConfigurationException {
-    configuration.setCloudProjectName(projectSelector.getText());
-    configuration.setCloudSdkExecutablePath(
-        CloudSdkUtil.toExecutablePath(cloudSdkDirectoryField.getText()));
+    if (StringUtil.isEmpty(cloudSdkDirectoryField.getText())
+        || !CloudSdkUtil.containsCloudSdkExecutable(cloudSdkDirectoryField.getText())) {
+      throw new RuntimeConfigurationError(
+          GctBundle.message("appengine.cloudsdk.location.missing.warning.message"));
```

Comment:
Those RuntimeConfigurationError exceptions get routed by the IJ to be displayed in the UI to the user. It's their mechanism for showing errors/warnings to the user when saving a panel with "bad" data. Not sure if your comment still applies?